app/src/main/java/com/cdv/pokemongo/ui/composables/PokemonImage.kt

```diff
@@ -0,0 +1,43 @@
+package com.cdv.pokemongo.ui.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.cdv.pokemongo.ui.theme.Green40
+
+@Composable
+fun PokemonImage(
+        modifier : Modifier = Modifier,
+        color : Color = Green40,
+        url : String = "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/35.png"
+){
```

Comment:
zrobiłbym to pole wymagane, a przykładowy link przekazał do preview, żeby gdzieś przez przypadek nie wyświetlić Clefairy 😅

Reply:
jasne, na ten moment było zostawione bardziej dla naszego testowania na szybko żeby linków nie szukać ale zaraz wywale